deploy/src/erc_to_erc/foreign.js

```diff
@@ -40,154 +87,102 @@ async function deployForeign() {
   const storageValidatorsForeign = await deployContract(EternalStorageProxy, [], {
     from: DEPLOYMENT_ACCOUNT_ADDRESS,
     network: 'foreign',
-    nonce: foreignNonce
+    nonce
   })
-  foreignNonce++
+  nonce++
   console.log('[Foreign] BridgeValidators Storage: ', storageValidatorsForeign.options.address)
 
   console.log('\ndeploying implementation for foreign validators')
   const bridgeValidatorsForeign = await deployContract(BridgeValidators, [], {
     from: DEPLOYMENT_ACCOUNT_ADDRESS,
     network: 'foreign',
-    nonce: foreignNonce
+    nonce
   })
-  foreignNonce++
+  nonce++
   console.log(
     '[Foreign] BridgeValidators Implementation: ',
     bridgeValidatorsForeign.options.address
   )
 
   console.log('\nhooking up eternal storage to BridgeValidators')
-  const upgradeToBridgeVForeignData = await storageValidatorsForeign.methods
-    .upgradeTo('1', bridgeValidatorsForeign.options.address)
-    .encodeABI({ from: DEPLOYMENT_ACCOUNT_ADDRESS })
-  const txUpgradeToBridgeVForeign = await sendRawTxForeign({
-    data: upgradeToBridgeVForeignData,
-    nonce: foreignNonce,
-    to: storageValidatorsForeign.options.address,
-    privateKey: deploymentPrivateKey,
+  await upgradeProxy({
```

Comment:
`upgradeProxy` in `src/deploymentUtils.js` calls `sendRawTxHome` and it means that `HOME_DEPLOYMENT_GAS_PRICE` will be used for the transaction in the Foreign chain. Don't think that it is correct. The same is applicable for other new function.

Reply:
Good catch! Thank you. Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/156/commits/f022bed9d19ac9126a56c8f4c5e272dfed66bdc5